packages/app/src/app/pages/common/Navigation/Notifications/notifications/TeamInvite.tsx

```diff
@@ -1,86 +1,90 @@
-import React from 'react';
-
-import { Mutation } from 'react-apollo';
-import history from 'app/utils/history';
-import { teamOverviewUrl } from '@codesandbox/common/lib/utils/url-generator';
 import track from '@codesandbox/common/lib/utils/analytics';
-import { inject, hooksObserver } from 'app/componentConnectors';
+import { teamOverviewUrl } from '@codesandbox/common/lib/utils/url-generator';
+import React, { FunctionComponent } from 'react';
+import { Mutation } from 'react-apollo';
 
-import { NotificationImage as Image } from '../elements';
-import { Container, Buttons, Button, W } from './elements';
+import { useOvermind } from 'app/overmind';
 import {
   REJECT_TEAM_INVITATION,
   ACCEPT_TEAM_INVITATION,
-} from '../../../../Dashboard/queries';
+} from 'app/pages/Dashboard/queries';
+import history from 'app/utils/history';
+
+import { NotificationImage as Image } from '../elements';
+
+import { Container, Buttons, Button, W } from './elements';
 
-interface Props {
+type Props = {
   read: boolean;
   teamId: string;
   teamName: string;
   inviterName: string;
   inviterAvatar: string;
-}
+};
+export const TeamInvite: FunctionComponent<Props> = ({
+  read,
+  teamId,
+  teamName,
+  inviterName,
+  inviterAvatar,
+}) => {
+  const {
+    actions: { notificationAdded },
+  } = useOvermind();
+
+  return (
+    <div>
+      <Container read={read}>
+        <Image src={inviterAvatar} />
+
+        <div>
+          <W>{inviterName}</W> invites you to join team <W>{teamName}</W>
+        </div>
+      </Container>
+
+      {!read && (
+        <Buttons>
+          <Mutation
+            variables={{ teamId }}
+            mutation={REJECT_TEAM_INVITATION}
+            refetchQueries={['RecentNotifications']}
+            onCompleted={() => {
+              track('Team - Invitation Rejected');
+              notificationAdded({
+                message: `Rejected invitation to ${teamName}`,
+                type: 'success',
```

Comment:
@christianalfoni It seems like `message` is renamed to `title`, `type` is renamed to `notificationType` _**and**_ a value (`number`) for `timeAlive` needs to be provided.
https://github.com/codesandbox/codesandbox-client/blob/cb965d440eb569d569e97ded524d54d142f241ad/packages/app/src/app/overmind/actions.ts#L28-L38

Any advice what I should do with the `timeAlive` value? 🤔
Should I set a default value or should I provide a value for them? 🤔

Reply:
Actually, since we now have more control I would suggest adding a new action called: `rejectTeamInvitation`. This action does the tracking and uses the `effects.notificationsToast.success(...)` instead. This cleans up the component a bit and we see more of what is going on when this scenario occurs :)

But yeah, would be awesome if you do a search for the usage of this `notificationAdded`, it is really deprecated. But yeah, for this specific scenario... new action :)